pymc_experimental/tests/model/test_marginal_model.py

```diff
@@ -251,6 +256,26 @@ def test_marginalized_change_point_model_sampling(disaster_model):
     )
 
 
+@pytest.mark.slow
+@pytest.mark.filterwarnings("error")
+def test_unmarginalized_basic(disaster_model):
```

Comment:
I would like a simpler model where we can assert the actual logps are what's expected